test/services/fs-service.test.ts

```diff
@@ -73,6 +73,30 @@ describe('fs-service', () => {
       expect(fs.existsSync(filePath)).to.eq(true)
     })
 
+  test
+    .do(() => {
+      fs.mkdirSync(path.resolve(tempDirHelper.tmpDir, defaultBundleName))
+      fs.mkdirSync(path.resolve(tempDirHelper.tmpDir, defaultBundleName, 'aux'))
```

Comment:
This could be replaced with just one line:
`fs.mkdirSync(path.resolve(tempDirHelper.tmpDir, defaultBundleName, 'aux'), { recursive: true })`

Reply:
done